packages/inference/src/providers/together.ts

```diff
@@ -54,6 +52,7 @@ export const TOGETHER_SUPPORTED_MODEL_IDS: ProviderMapping<TogetherId> = {
 		"scb10x/llama-3-typhoon-v1.5x-70b-instruct-awq": "scb10x/scb10x-llama3-typhoon-v1-5x-4f316",
 	},
 	"text-generation": {
+		"meta-llama/Llama-2-70b-hf": "meta-llama/Llama-2-70b-hf",
 		"meta-llama/Meta-Llama-3-8B": "meta-llama/Meta-Llama-3-8B",
```

Comment:
Can we remove this one as well?
It's not properly supported (errors out on inference)
```suggestion
```